zipline/data/us_equity_pricing.py

```diff
@@ -0,0 +1,109 @@
+# Copyright 2015 Quantopian, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from bcolz import ctable
+import pandas as pd
+
+from sid import string_types
+
+
+class NoDataOnDate(Exception):
+    """
+    Raised when a spot price can be found for the sid and date.
+    """
+    pass
+
+
+class BcolzDailyBarSpotReader(object):
+    """
+    Reads inividual price moments from the daily bcolz file format shared
+    with pipeline.loaders.equity_pricing_loader.BcolzOHLCVReader
```

Comment:
Did you rename the class elsewhere in this PR?  If not then I don't think this is the right name.